### PR TITLE
added skale chain version

### PR DESCRIPTION
**Pull request by ebruengwall-ant**

### CODE REVIEW

File: validators/schain-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+ 4. Check-out to sgx version tag: `git checkout tags/1.58.7-develop.1`
+ 5. Make sure `image` is skalenetwork/sgxwallet:1.58.7-develop.1 in docker-compose:
+ 6. [Recover from back up] (https://skale.network/docs/documentation/sgxwallet/docs/backup-procedure)
+ 
+#### Step 2.2 Install SKALE Node CLI
+
+#### Download the SKALE Node CLI binary
+
+Make sure th `VERSION_NUM` is the 1.1.0-beta.3
+For `RELEASE` parameter use the `develop` CLI versions use `develop` , for the `beta` CLI versions use `beta` , for the `stable` CLI versions use `stable`
```
Comment (badrogger) [inline validators/schain-upgrade.md]: Where is release parameter in the following command ?

File: validators/schain-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+Consensus: the build introduces BLAKE3 hash and consensus DB changes (for better performance). Should not impact update process
+
+#### Important
+
+1. Validators need to update docker-compose version on their machines to 1.27.4, because we started using new docker-compose syntax (cpu_shares functionality in particular).
+2. Validators need to remove .skale/node_data/skale.db before skale node update , because we updated db schema and until we have the migration ready we need to fast through this step by removing the folder.
+3. Resource allocation file should be re-created on all nodes - this should be done automatically during node update :
+    Why resource allocation should be updated?
+        i. In this update, we’re adding CPU and memory limits for the IMA container (this will also affect sChain container allocation)
```
Comment (badrogger) [inline validators/schain-upgrade.md]: List is displayed in one line.

File: validators/schain-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+**SKALE Manager version**: 1.6.0-develop.7
+
+**Skaled version**:  1.46-develop.63
+
+**Skale Admin version**: 0.17.0-beta.1
+
+**Transaction Manager version**: 0.5.0-beta.1
+
+**Skale Sla version**: 1.0.0-develop.0
```
Comment (badrogger) [inline validators/schain-upgrade.md]: Should be 1.0.2-beta.1

File: validators/schain-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+**SGX version: sgxwallet**: 1.58.7-develop.1
+
+**SKALE Manager version**: 1.6.0-develop.7
+
+**Skaled version**:  1.46-develop.63
+
+**Skale Admin version**: 0.17.0-beta.1
+
+**Transaction Manager version**: 0.5.0-beta.1
```
Comment (badrogger) [inline validators/schain-upgrade.md]: Should be 1.0.0-develop.8

File: validators/schain-upgrade.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+**Node CLI version**: 1.1.0-beta.3
+
+**SGX version: sgxwallet**: 1.58.7-develop.1
+
+**SKALE Manager version**: 1.6.0-develop.7
+
+**Skaled version**:  1.46-develop.63
+
+**Skale Admin version**: 0.17.0-beta.1
```
Comment (badrogger) [inline validators/schain-upgrade.md]: Should be  1.1.0-develop.17

File: validators/schain-upgrade.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+**Validator CLI version**: 1.2.0-beta.1
+
+**Node CLI version**: 1.1.0-beta.3
+
+**SGX version: sgxwallet**: 1.58.7-develop.1
+
+**SKALE Manager version**: 1.6.0-develop.7
+
+**Skaled version**:  1.46-develop.63
```
Comment (badrogger) [inline validators/schain-upgrade.md]: Should be 2.0.0-beta.0

**Review by badrogger — CHANGES_REQUESTED** [review]

File: validators/schain-upgrade.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+**Skale Admin version**: 0.17.0-beta.1
+
+**Transaction Manager version**: 0.5.0-beta.1
+
+**Skale Sla version**: 1.0.0-develop.0
+
+**Skale Bounty version**: 0.9.3-develop.0
+
```
Comment (gannakulikova) [inline validators/schain-upgrade.md:115]: @badrogger @dmitry-tk  it seems that we need **1.1.0-stable.0** here?

**Review by christineRPM — DISMISSED** [review]

File: validators/fuji-versions.md [hunk 8]
```diff
@@ -1,9 +1,42 @@
...
+
+**Node CLI version**: 1.1.0-beta.4
+
+**SGX version: sgxwallet**: 1.59.1-stable.6
+
+**SKALE Manager version**: 1.6.2-develop.0
+
+**Skaled version**: 2.0.4-develop.3
+
+**Skale Admin version**: 1.1.0-develop.17
```
[the remark this answers is not in the document]
Comment (ebruengwall-ant) [reply validators/fuji-versions.md]: Are we good with this? @badrogger

File: validators/schain-upgrade.md (hunk 5 above)
Comment (ebruengwall-ant) [reply validators/schain-upgrade.md]: @badrogger Can you please update the old versions?

File: validators/schain-upgrade.md (hunk 1 above)
Comment (ebruengwall-ant) [reply validators/schain-upgrade.md]: yeah, it's an old comment for the digital ocean. We can remove that line.

**Review by christineRPM — DISMISSED** [review]

**Review by gannakulikova — DISMISSED** [review]

**Review by ebruengwall-ant — COMMENTED** [review]
all good

File: validators/schain-upgrade.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+**Transaction Manager version**: 1.0.0-develop.8
+
+**Skale Sla version**: 1.0.0-develop.0
+
+**Skale Bounty version**: 1.1.0-stable.0
+
+**docker-lvmpy**: 1.0.1
+
+**skale-admin**: 1.1.0-develop.21
+
```
[the remark this answers is not in the document]
Comment (ebruengwall-ant) [reply validators/schain-upgrade.md:109]: is it for skale admin?

[the remark this answers is not in the document]
Comment (gannakulikova) [reply validators/schain-upgrade.md:109]: yes https://skalelabs.atlassian.net/wiki/spaces/SKALE/pages/1195311113/TestNet+Upgrade+-+November+December+2020+Schains+and+IMA+Beta#Release-Candidate-Versions---Schains-TestNet-Update---December--30%2FJanuary-4

**Review by gannakulikova — APPROVED** [review]

**Pull request merged**